assets/dex/planetswap.json

```diff
@@ -0,0 +1,22 @@
+{
+    "metadata": {
+        "label": "planetswap",
+        "category": "DEX",
```

Comment:
@ohld could you please check that category is correct in this case (may be it is gaming?)

Reply:
I double checked: this project doesn't have any link to a swap or to a website that claims that actually have a decetralized exchange. I think, they just issuing a token for now. Let's label them as `category: gaming`. Once they release a decentralized exchange on TON, we'd be happy to update the category.